config/locales/en.yml

```diff
@@ -1238,37 +1219,24 @@ en:
       export_csv: "Export to CSV"
       email: "Email"
       approve: "Approve"
-      reject: "Reject"
-      delete: "Delete"
       waiting_list: "Waiting List"
       approved_registrations: "Approved registrations"
-      deleted_registrations: "Deleted registrations"
-      non_competing: "Non-competing registrations"
       not_paid: "Not paid"
       registered:
         without_stripe: "Registered on"
         with_stripe: "Paid on"
       #context: date is the registration creation date, eg: "2016-12-01 16:18:56 UTC"
       payment_requested_on: "Payment requested on: %{date}"
       payment_completed_on: "Payment completed on: %{date}"
-      #context: Counts how many parallel registrations in the same Series there are. Will display a number behind this label.
-      series_registrations: "Series registrations"
```

Comment:
Do we show parallel Series registrations in the new system?

Reply:
no, I'm not sure what this did, but we don't allow registering for both competitions in a series

Reply:
added todo